src/types/networks.ts

```diff
@@ -1,10 +1,4 @@
-import { NETWORKS, SIGNET, REGTEST } from "../networks";
+import { Network } from "../networks";
 
-const NETWORKS_ENUM = {
-  ...NETWORKS,
-  REGTEST,
-  SIGNET,
-} as const;
-
-type NETWORKS_KEYS = keyof typeof NETWORKS_ENUM;
-export type BitcoinNetwork = (typeof NETWORKS_ENUM)[NETWORKS_KEYS];
+type NETWORKS_KEYS = keyof typeof Network;
+export type BitcoinNetwork = (typeof Network)[NETWORKS_KEYS];
```

Comment:
probably don't even need this anymore tbh.

Reply:
It was left for backwards compatibility purpose. Removing it would be breaking.

Reply:
I guess I just figured since we're assumign this whole thing was going to be breaking we could get away with it, but not a huge deal either way.